docs/accessibility-test-plan.md

```diff
@@ -0,0 +1,48 @@
+# Manual Test Plan
+
+## KEYBOARD NAVIGATION
```

Comment:
Editorial nit: the all-caps on these heading isn't consistent with the rest of our docs. We generally use Title Case (or Sentence case; we should pick one and stick to it).